crypt-gpgme.c

```diff
@@ -2097,13 +2097,13 @@ static int pgp_gpgme_extract_keys (gpgme_data_t keydata, FILE** fp, int dryrun)
       strftime (date, sizeof (date), "%Y-%m-%d", localtime (&tt));
 
       if (!more)
-        fprintf (*fp, "%s %5.5s %d/%8s %s %s\n", more ? "sub" : "pub",
+        fprintf (*fp, "pub %5.5s %d/%8s %s %s\n",
                  gpgme_pubkey_algo_name (subkey->pubkey_algo), subkey->length,
                  shortid, date, uid->uid);
       else
-        fprintf (*fp, "%s %5.5s %d/%8s %s\n", more ? "sub" : "pub",
+        fprintf (*fp, "sub %5.5s %d/%8s %s\n",
                  gpgme_pubkey_algo_name (subkey->pubkey_algo), subkey->length,
-                 shortid, date);      
+                 shortid, date);
       subkey = subkey->next;
       more = 1;
```

Comment:
For these two `fprintf`s, we know whether `more` is true or false, so we can drop the first parameter.
The "pub" and "sub" would only need to be params if they were being translated.